support/SubmitScripts/Frontera.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#SBATCH -J MyJobName       # Job name
+#SBATCH -o MyJobName.o%j   # Name of stdout output file
+#SBATCH -e MyJobName.e%j   # Name of stderr error file
+#SBATCH -p normal          # Queue (partition) name
+#SBATCH -N 2               # Total # of nodes
+#SBATCH -n 2               # Total # of tasks, must be number of nodes
+#SBATCH -t 00:30:00        # Run time (hh:mm:ss)
+# #SBATCH --mail-type=all  # Send email at begin and end of job
+# #SBATCH --mail-user=username@tacc.utexas.edu
+# #SBATCH -A account       # Project/Allocation name (req'd if you have more than 1)
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# UPDATE THESE!!!
+export SPECTRE_EXECUTABLE=./bin/ParallelInfo
+export SPECTRE_INPUT_FILE=./Test.yaml
+
+# Print out diagnostic info
+module list 2>&1
+pwd
+date
+
+echo ""
+echo ""
+echo ""
+
+export IBRUN_TASKS_PER_NODE=1
+
+if [ -f ${SPECTRE_EXECUTABLE} ]; then
+    if [ -f ${SPECTRE_INPUT_FILE} ]; then
+        ibrun -n ${SLURM_JOB_NUM_NODES} \
+            ${SPECTRE_EXECUTABLE} ++ppn 55 \
+            --input-file ${SPECTRE_INPUT_FILE} > SpECTRE.out 2>&1
```

Comment:
what goes into SpECTRE.out that does not go into the `-o` and `-e` `SBATCH` options?

Reply:
As is nothing from the spectre output goes into `-o` and `-e`. I think this is what SpEC did, though I'm not sure I see any value in it. I do like having all the error and standard output in one file. What do you think of dumping all executable output into `-o`?

Reply:
that would be fine with me

Reply:
Okay, updated and squashed